phpcs.xml.dist

```diff
@@ -133,8 +131,6 @@
 
     <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint">
         <properties>
-            <!-- Requires PHP 7.2 -->
-            <property name="enableObjectTypeHint" value="false" />
```

Comment:
In this case, the comment above was always accurate (in contrast to the `ParameterTypeHint` rule above) and we're just removing this because the library requires 7.2+, correct?

Reply:
That is correct, we just didn't touch this when dropping support for PHP < 7.2.